lib/auth_helper.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:async';
```

Comment:
I prefer auth_service than auth_helper but It is a matter of personal preference.